tools/file_packager.py

```diff
@@ -914,6 +914,10 @@ def generate_js(data_target, data_files, metadata):
           for (var chunkId = 0; chunkId < chunkCount; chunkId++) {
             var getRequest = packages.get(`package/${packageName}/${chunkId}`);
             getRequest.onsuccess = function(event) {
+              if (!event.target.result) {
+                errback(new Error("CachedPackageNotFound for: " + packageName));
```

Comment:
Can you use backticks to format this new error message?

Reply:
Fixed